src/cli.rs

```diff
@@ -32,6 +32,14 @@ pub(crate) struct Cli {
     #[arg(short = 'l', long = "list")]
     pub list: bool,
 
+    /// Edit custom patch with `EDITOR`
+    #[arg(long = "edit-patch")]
+    pub edit_patch: bool,
+
+    /// Edit custom page with `EDITOR`
+    #[arg(long = "edit-page")]
+    pub edit_page: bool,
```

Comment:
The defaults from clap already replace `_` with `-`, so this is equivalent:
```suggestion
    /// Edit custom patch with `EDITOR`
    #[arg(long)]
    pub edit_patch: bool,

    /// Edit custom page with `EDITOR`
    #[arg(long)]
    pub edit_page: bool,
```